.storybook/preview.js

```diff
@@ -5,11 +5,12 @@ import { MockedProvider } from '@apollo/client/testing'
 // happo support
 import 'happo-plugin-storybook/register'
 
-import 'uswds/dist/css/uswds.css'
-import '../src/initIcons'
-import '../src/styles/index.scss'
+import 'styles/index.scss'
+// #TODO these imports seem unneccessary can we delete them?
 // SFDS
-import '../src/styles/sfds/index.scss'
+//import 'styles/sfds/index.scss'
+// import '../src/styles/sfds/index.scss'
+//import 'initIcons'
```

Comment:
I'm going to remove these commented out lines, I don't think we need them. Objections?

Reply:
i was going to leave them until every issue was resolved, just to make sure it was okay to delete

Reply:
update: we needed initIcons!